tests/runner.py

```diff
@@ -510,7 +510,7 @@ def get_emcc_args(self, main_file=False):
   # Build JavaScript code from source code
   def build(self, src, dirname, filename, main_file=None,
             additional_files=[], libraries=[], includes=[], build_ll_hook=None,
-            post_build=None, js_outfile=True):
+            post_build=None, js_outfile=True, extra_flags=[]):
```

Comment:
why do we need `extra_flags`? I think we should be able to use `self.emcc_args` (which is then processed and returned in `self.get_emcc_args` later down), unless I'm missing something.